ui/src/app/components/genomics-extraction-table.tsx

```diff
@@ -64,9 +73,70 @@ const formatDuration = (durationMoment) => {
   return (hours > 0 ? hours + ' hr, ' : '') + minStr;
 };
 
+const GenomicsExtractionMenu = ({job, workspace}) => {
+  return <PopupTrigger
+    side='bottom-left'
+    closeOnClick
+    content={
+     <React.Fragment>
+       <MenuItem
+           faIcon={faLocationCircle}
+           disabled
+           onClick={() => {}}
+       >
+        View Path
+       </MenuItem>
+       <MenuItem
+           faIcon={faClipboard}
+           disabled
+           onClick={() => {}}
+       >
+         Copy Snippet
+       </MenuItem>
+       <hr/>
+       <MenuItem
+           faIcon={faBan}
+           disabled={job.status !== TerraJobStatus.RUNNING || !WorkspacePermissionsUtil.canWrite(workspace.accessLevel)}
+           onClick={() => {
+             dataSetApi().abortExtract(workspace.namespace, workspace.id, job.genomicExtractionJobId);
+           }}
+           tooltip={
+             job.status !== TerraJobStatus.RUNNING
+               ? 'Extraction job is not currently running'
+               : !WorkspacePermissionsUtil.canWrite(workspace.accessLevel)
+                 ? 'You do not have permission to modify this workspace'
+                 : ''
+           }
+       >
+         Abort Extraction
+       </MenuItem>
+       <hr/>
+       <MenuItem
+         faIcon={faTrash}
+         disabled
+         onClick={() => {}}
+       >
+         Delete Extract
+       </MenuItem>
+     </React.Fragment>
+    }
+  >
+    <FontAwesomeIcon
+        icon={faEllipsisV}
+        style={{
+          color: colors.accent,
+          fontSize: '.7rem',
```

Comment:
Ah.. I didn't realize you could use font size to control this. Thanks

Reply:
This is apparently the fontawesome-y way to control it.